app/serializers/about_serializer.rb

```diff
@@ -61,4 +71,15 @@ def can_see_about_stats
   def can_see_site_contact_details
     scope.can_see_site_contact_details?
   end
+
+  def render_redesigned_about_page?
+    return false if !scope.user
+
+    return @render_redesigned_about_page if @render_redesigned_about_page != nil
```

Comment:
Instead of this, below you can just memoize:

```ruby
@render_redesigned_about_page ||= scope.user.in_any_groups?(SiteSetting.experimental_redesigned_about_page_groups_map)
```

Reply:
I just removed the memorization entirely because it's really micro optimization. The method is only called twice and `in_any_groups?` already memoizes the user's group IDs so there's no db query on each call.